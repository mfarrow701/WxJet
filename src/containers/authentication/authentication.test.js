import React from 'react';
import {shallow} from 'enzyme';
import Authentication from './authentication';

describe('Authentication container', () => {
    let component;

    const renderWrapper = () => {
        component = shallow(<Authentication/>);
    };

    beforeEach(() => {
        renderWrapper();
    });

    it('should render the authentication container after load', () => {
        expect(component).toMatchSnapshot();
        expect(component.find('.Authentication h1').text()).toEqual('Not Found!');
    });
});
