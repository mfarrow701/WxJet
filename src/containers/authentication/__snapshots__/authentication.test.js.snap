// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Authentication container should render the authentication container after load 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Authentication />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "autoComplete": "on",
      "children": <Unknown>
        <img
                alt=""
                src="logo.svg"
        />
        <input
                name="email"
                onChange={[Function]}
                placeholder="Email"
                style={
                        Object {
                                "borderBottom": "4px solid red",
                              }
                }
                type="email"
                value=""
        />
        <input
                name="password"
                onChange={[Function]}
                placeholder="Password"
                style={
                        Object {
                                "borderBottom": "4px solid red",
                              }
                }
                type="password"
                value=""
        />
        <button
                disabled={true}
                type="submit"
        >
                Sign in
        </button>
        <FormErrors
                formErrors={
                        Object {
                                "email": "",
                                "password": "",
                              }
                }
        />
        <p
                onClick={[Function]}
        >
                Not registered?
        </p>
</Unknown>,
      "className": "Authentication",
      "onSubmit": [Function],
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <img
            alt=""
            src="logo.svg"
/>,
          <input
            name="email"
            onChange={[Function]}
            placeholder="Email"
            style={
                        Object {
                                    "borderBottom": "4px solid red",
                                  }
            }
            type="email"
            value=""
/>,
          <input
            name="password"
            onChange={[Function]}
            placeholder="Password"
            style={
                        Object {
                                    "borderBottom": "4px solid red",
                                  }
            }
            type="password"
            value=""
/>,
          <button
            disabled={true}
            type="submit"
>
            Sign in
</button>,
          <FormErrors
            formErrors={
                        Object {
                                    "email": "",
                                    "password": "",
                                  }
            }
/>,
          <p
            onClick={[Function]}
>
            Not registered?
</p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "",
            "src": "logo.svg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "name": "email",
            "onChange": [Function],
            "placeholder": "Email",
            "style": Object {
              "borderBottom": "4px solid red",
            },
            "type": "email",
            "value": "",
          },
          "ref": null,
          "rendered": null,
          "type": "input",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "name": "password",
            "onChange": [Function],
            "placeholder": "Password",
            "style": Object {
              "borderBottom": "4px solid red",
            },
            "type": "password",
            "value": "",
          },
          "ref": null,
          "rendered": null,
          "type": "input",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Sign in",
            "disabled": true,
            "type": "submit",
          },
          "ref": null,
          "rendered": "Sign in",
          "type": "button",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "formErrors": Object {
              "email": "",
              "password": "",
            },
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Not registered?",
            "onClick": [Function],
          },
          "ref": null,
          "rendered": "Not registered?",
          "type": "p",
        },
      ],
      "type": Symbol(react.fragment),
    },
    "type": "form",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "autoComplete": "on",
        "children": <Unknown>
          <img
                    alt=""
                    src="logo.svg"
          />
          <input
                    name="email"
                    onChange={[Function]}
                    placeholder="Email"
                    style={
                              Object {
                                        "borderBottom": "4px solid red",
                                      }
                    }
                    type="email"
                    value=""
          />
          <input
                    name="password"
                    onChange={[Function]}
                    placeholder="Password"
                    style={
                              Object {
                                        "borderBottom": "4px solid red",
                                      }
                    }
                    type="password"
                    value=""
          />
          <button
                    disabled={true}
                    type="submit"
          >
                    Sign in
          </button>
          <FormErrors
                    formErrors={
                              Object {
                                        "email": "",
                                        "password": "",
                                      }
                    }
          />
          <p
                    onClick={[Function]}
          >
                    Not registered?
          </p>
</Unknown>,
        "className": "Authentication",
        "onSubmit": [Function],
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            <img
              alt=""
              src="logo.svg"
/>,
            <input
              name="email"
              onChange={[Function]}
              placeholder="Email"
              style={
                            Object {
                                          "borderBottom": "4px solid red",
                                        }
              }
              type="email"
              value=""
/>,
            <input
              name="password"
              onChange={[Function]}
              placeholder="Password"
              style={
                            Object {
                                          "borderBottom": "4px solid red",
                                        }
              }
              type="password"
              value=""
/>,
            <button
              disabled={true}
              type="submit"
>
              Sign in
</button>,
            <FormErrors
              formErrors={
                            Object {
                                          "email": "",
                                          "password": "",
                                        }
              }
/>,
            <p
              onClick={[Function]}
>
              Not registered?
</p>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "",
              "src": "logo.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "name": "email",
              "onChange": [Function],
              "placeholder": "Email",
              "style": Object {
                "borderBottom": "4px solid red",
              },
              "type": "email",
              "value": "",
            },
            "ref": null,
            "rendered": null,
            "type": "input",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "name": "password",
              "onChange": [Function],
              "placeholder": "Password",
              "style": Object {
                "borderBottom": "4px solid red",
              },
              "type": "password",
              "value": "",
            },
            "ref": null,
            "rendered": null,
            "type": "input",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Sign in",
              "disabled": true,
              "type": "submit",
            },
            "ref": null,
            "rendered": "Sign in",
            "type": "button",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "formErrors": Object {
                "email": "",
                "password": "",
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Not registered?",
              "onClick": [Function],
            },
            "ref": null,
            "rendered": "Not registered?",
            "type": "p",
          },
        ],
        "type": Symbol(react.fragment),
      },
      "type": "form",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
